src/code/z_vr_box.c

```diff
@@ -210,17 +210,17 @@ void Skybox_Setup(GameState* gameState, SkyboxContext* skyboxCtx, s16 skyboxId)
             segment = (void*)ALIGN8((uintptr_t)segment + size);
             DmaMgr_SendRequest0(skyboxCtx->paletteStaticSegment, SEGMENT_ROM_START(d2_fine_pal_static), size);
 
-            skyboxCtx->primR = 145;
-            skyboxCtx->primG = 120;
-            skyboxCtx->primB = 155;
+            skyboxCtx->prim.r = 145;
+            skyboxCtx->prim.g = 120;
+            skyboxCtx->prim.b = 155;
 
-            skyboxCtx->envR = 40;
-            skyboxCtx->envG = 0;
-            skyboxCtx->envB = 40;
+            skyboxCtx->env.r = 40;
+            skyboxCtx->env.g = 0;
+            skyboxCtx->env.b = 40;
 
             // Inverted Stone Tower Temple and Inverted Stone Tower
             if ((play->sceneId == SCENE_F41) || (play->sceneId == SCENE_INISIE_R)) {
-                skyboxCtx->rotZ = 3.15f;
+                skyboxCtx->rot.z = 3.15f;
```

Comment:
Not even 3.14

Reply:
Reminded me of this image lmao
![image](https://user-images.githubusercontent.com/7416381/222172423-1c86ed8a-f53c-4d26-82b7-73f115fee957.png)